doc/user/content/get-started.md

```diff
@@ -12,54 +12,35 @@ aliases:
   - /install/
 ---
 
-[//]: # "TODO(morsapaes) Once we're GA, add details about signing up and logging
-into a Materialize account"
+This guide walks you through getting started with Materialize.
 
-This guide walks you through getting started with Materialize, covering:
+- First, you'll sign up for a free Materialize trial account and prepare your environment.
+- Next, you'll connect a sample streaming data source and explore the unique value of Materialize: computing real-time results over fast-changing data.
+- Finally, you'll simulate a failure to see active replication in action.
```

Comment:
In the previous version, the intro served more the purpose of quickly highlighting some key concepts, as well as features that users frequently ask about, like joins and time windows (_i.e._ temporal filters, not window functions). This feels slightly less useful.

Reply:
Yeah, this new version summarizes the flow and then mentions some concepts you'll learn through the experience, but I also don't love it. I'll iterate on this, but the previous version didn't really signal anything about the "wow" of the product, which I'd like to find a way to keep, as that's the primary experience we want users to take-away.